WORKSPACE

```diff
@@ -257,6 +257,10 @@ maven_install(
         # Any two platforms to ensure that it doesn't work _only_ under the host operating system
         "com.google.protobuf:protoc:exe:linux-x86_64:3.21.12",
         "com.google.protobuf:protoc:exe:osx-aarch_64:3.21.12",
+        # https://github.com/bazelbuild/rules_jvm_external/issues/917
+        # androidx core-testing POM has "exclusion" for "byte-buddy" but it should be downloaded as mockito-core dependency.
+        "org.mockito:mockito-core:jar:3.3.3",
```

Comment:
Since `jar` is the default, it's usually omitted.

Reply:
Yep yep, it's just a convention in our large Android monorepo to see what deps are JVM and what are Android AARs I think it's fine for purpose of this regression test

Reply:
Agree.